starters/e2e/e2e.qrl.e2e.ts

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from "@playwright/test";
+test("should update counter without uncaught promises", async ({ page }) => {
+  await page.goto("/e2e/qrl");
+  page.on("pageerror", (err) => expect(err).toEqual(undefined));
+  page.on("console", (msg) => {
+    if (msg.type() === "error") {
+      expect(msg.text()).toEqual(undefined);
+    }
+  });
+  const button = page.locator("#inner-computed-button");
+  await expect(button).toContainText("Click Me 0");
+
+  // 先执行点击操作
```

Comment:
comments in English only please :)
Here I don't think a comment is needed actually